boms/integration-tests/src/test/java/com/google/cloud/BomContentTest.java

```diff
@@ -66,9 +70,17 @@ public void testLibrariesBom() throws Exception {
 
   private void checkBom(Path bomPath) throws Exception {
     Bom bom = Bom.readBom(bomPath);
+
+    // Sometimes the artifacts are not yet available in Maven Central and only available in local
+    // Maven repository. Use
+    boolean enableMavenCentralAvailabilityCheck =
```

Comment:
The variable name is the opposite of the property name, can we keep them consistent? Either `disableMavenCentralCheck` or `enableMavenCentralAvailabilityCheck`?

Reply:
Updated.